packages/dds/tree/tsconfig.base.json

```diff
@@ -0,0 +1,17 @@
+{
+	"extends": "@fluidframework/build-common/ts-common-config.json",
```

Comment:
Any reason not to use the new shared configs here?

Reply:
When I look in @fluidframework/build-common that is the only config file. Workspace needs an @fluidframework/build-common update?

Reply:
We used relative paths so we don't have to publish build-common all the time. Look at merge-tree or other packages for the new pattern. Very open to changes to the pattern in the future if you have feedback on it.

See also https://github.com/microsoft/FluidFramework/pull/18152 but the pr description is not completely accurate. Especially about node10 and esnext. Oh, I was so naive then :)

Reply:
I've seen those and thought sure that must be temporary. That is far too many `../`. Is the plan not to publish then?
We can do a bit better. For now I would go with getting the settings right (as I have them). And then we can roll over to a nicer pattern of common definitions.
Won't fix for this PR.

Reply:
We can eventually publish but IMO relative paths are a lot better for in-repo infrastructure because it allows for instant application of changes throughout the release group. If we had not used this pattern when making these changes we would have had to publish at least three versions of the package.

We can still publish the package if/when things are settled and/or we want to make the changes more easily available to other release groups.

Reply:
Yes, having a way to consume instantly is good. I don't think we need to publish to resolve the general relative path issue. But I still think not for this change. A nice thing to do after; here keep changes and no changes very clear.